index.js

```diff
@@ -1,9 +1,17 @@
 'use strict';
 var got = require('got');
-var registryUrl = require('registry-url');
+var npmKeywordUrl = require('npm-keyword-url');
```

Comment:
> Its actually pretty useful.

Saying it's pretty useful, it's not a very useful statement. Some context, please. Why can you not use this module?